src/loaders/vorbis.c

```diff
@@ -920,7 +920,7 @@ struct stb_vorbis
    int channel_buffer_start;
    int channel_buffer_end;
 
-  // libxmp hack: decode work buffer (used in inverse_mdct and decode_residues)
+  // hack: decode work buffer (used in inverse_mdct and decode_residues)
```

Comment:
If these patches have been merged upstream or we're changing upstreams, the "libxmp hack" comments can be removed entirely. They aren't part of my upstream patches, they're just documentation I added here to make it clear that they haven't been merged yet. I don't think obfuscating this intent further is a good idea though.